app/templates/components/article-media-thumbnail.hbs

```diff
@@ -1,15 +1,15 @@
 {{! This wrapper is required so the chevron is positioned correctly }}
 <div class="article-media-thumbnail-image-wrapper">
 	{{#if isOgg}}
-		<a href="{{href}}">{{svg 'wds-icons-download' viewBox='0 0 24 24' class='icon'}}</a>
+		<a href="{{href}}">{{svg 'wds-icons-download' viewBox='0 0 24 24' class='wds-icon'}}</a>
 	{{else}}
 		<a href="{{href}}">
 			<img alt="{{title}}"
 				 src="{{thumbnailUrl}}"
 				 height="{{if forcedHeight forcedHeight height}}"
 				 width="{{if forcedWidth forcedWidth width}}">
 			{{#if isLinkedByUser}}
-				{{svg 'chevron-linked' viewBox='0 0 20 20' class='chevron'}}
+				{{svg 'wds-icons-menu-control' viewBox='0 0 12 12' class='wds-icon wds-icon-tiny wds-icons-tiny chevron'}}
```

Comment:
wds-icons-menu-control-tiny and remove viewBox

Reply:
what is the difference between css classes `wds-icon-tiny` and `wds-icons-tiny`? is the later one defined at all?